cpp/src/arrow/filesystem/gcsfs.h

```diff
@@ -232,7 +232,7 @@ class ARROW_EXPORT GcsFileSystem : public FileSystem {
 
   /// Create a GcsFileSystem instance from the given options.
   // TODO(ARROW-16884): make this return Result for consistency
```

Comment:
Could you remove this comment?

Reply:
Complete! Ref 4d6687ad52ac1734fab38e092ebc149fe87b09ce.